packages/grafana-ui/src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -7,6 +7,9 @@ import { getColorForTheme } from '@grafana/data';
 import { SeriesColorPickerPopover } from './SeriesColorPickerPopover';
 
 import { withTheme } from '../../themes/ThemeContext';
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { stylesFactory } from '../../themes';
```

Comment:
withTheme and stylesFactor can be imported with single import:)
import { withTheme, stylesFactory } from '../../themes';